packages/server/test/integration/cypress_spec.js

```diff
@@ -7,6 +7,7 @@ const Promise = require('bluebird')
 const electron = require('electron')
 const commitInfo = require('@cypress/commit-info')
 const Fixtures = require('@tooling/system-tests')
+const SystemTests = require('@tooling/system-tests/lib/system-tests')
```

Comment:
I think this `require` changes the process's working directory, which is why these tests are failing: https://app.circleci.com/pipelines/github/cypress-io/cypress/40774/workflows/983f4e5c-d12e-488f-9e43-c455632b2da0/jobs/1683656

The `process.cwd` call could be relocated, or `normalizeStdout` could be moved to a module that doesn't call `process.cwd`.

Reply:
Yep, was just looking into why this was failing. I'll make that refactor.

Ideally I'd have liked to have written this in `system-tests`, but it's quite hard to get the stdout to the correct state w/o a lot of stubbing, which system tests don't support for obvious reasons.

Reply:
Lol, hate to see it: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/system-tests.ts#L31. Always good to fix a TODO.

Reply:
Actually, problem is here: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/spec_helper.js (somewhere). I will find it.

Reply:
This is to blame for at least some of the problems: https://github.com/cypress-io/cypress/blob/develop/system-tests/lib/spec_helper.js#L83-L86

Reply:
I think we probably **don't** want all these side effects in `packages/server`. I'll just move the normalizeStdout function to a file, without any side effects.

Reply:
Ok, I did it.